script/run_test_suite.sh

```diff
@@ -31,11 +39,28 @@ if [[ "$system_hostname" =~ ^[0-9]+$ || "$shell_hostname" =~ ^[0-9]+$ ]] ; then
     exit 1
 fi
 
+if [ ! -z ${MC_TEST_SUITE_REPORT_COVERAGE+x} ]; then
+
+    echo "Removing old test coverage reports..."
+    rm -rf cover_db/ .coverage coverage.json
+
+    # Enable Perl's Devel::Cover
+    export HARNESS_PERL_SWITCHES='-MDevel::Cover=+ignore,t/,+ignore,\.t$'
+
+    # Enable Python's pytest coverage
+    PYTEST_ARGS="--cov=mediacloud/ --cov-report=term --cov-config=.coveragerc"
```

Comment:
- should this be `--cov=mediacloud/mediawords` (right now we are picking up the vendored `snowball` library)? 
- i think `--cov-report=term` is the default if you want to save some space
- we can save some file clutter by using `setup.cfg` instead of `.coveragerc` (though if we merge this first, i can make that change once the linting PR lands)

Reply:
Fixed some redundant arguments, thank you. Not sure why we'd ever need `setup.cfg` because Media Cloud code is not a Python module (or could / should be made into one as a whole).

Reply:
Hrm... I think you're thinking of `setup.py`, which is different, but also a good idea!  `setup.cfg` is an attempt to standardize where configuration goes for various tools to prevent a proliferation of dotfiles.  See, for example, https://github.com/pymc-devs/pymc3/blob/master/setup.cfg

Reply:
OK, should we put `setup.cfg` it in `mediacloud/` or in the root directory?

Reply:
I've tried to move coverage configuration properties to `setup.cfg` (which is a good idea) but it didn't work. It seems that somehow `pytest` is able to find and `pytest`-related properties from `setup.cfg` but not `coverage` ones - `[coverage:report]`, `[tool:coverage:report]`, `[report]` and other variations that I've tried weren't being read. So, after 30 minutes I just gave up.